src/scripts/game.js

```diff
@@ -196,6 +211,22 @@ class Game {
     return randomPos;
   }
 
+  flowerPosGenerator() {
+    let positions = [];
+    let minDistance = 80;
+    while (positions.length < Game.NUM_FLOWERS) {
+      let newPos = this.randomPosition();
+      let spreadOut = true;
+      for (let i = 0; i < positions.length; i++) {
+        if (Util.pointDistance(newPos, positions[i]) < minDistance) {
+          spreadOut = false;
+        }
+      }
```

Comment:
You're iterating through all the positions here every time.

Reply:
I shall use a break to exit the for loop if we find that spreadOut is true, so that we can more quickly generate a new pos to test.